### PR TITLE
Identifier

### PR DESCRIPTION
**Pull request by pkarimib**

### CODE REVIEW

File: examples/videostream-cli/cli.py [hunk 1]
```diff
@@ -133,6 +134,10 @@ def on_track(track):
     parser.add_argument("--verbose", "-v", action="count")
     parser.add_argument("--fps", type=int, help="fps you want to sample at")
     parser.add_argument("--save-dir", type=str, help="folder to save frames + latency data in")
+    parser.add_argument('--enable-prediction', action='store_true')
+    parser.add_argument("--output-fps", type=int, default=30, help="fps you want to save the video with")
```
Comment (vibhaa) [inline examples/videostream-cli/cli.py:138]: please shorten these two lines by making the help go on the next line.

Comment (vibhaa) [reply examples/videostream-cli/cli.py:138]: this and the next

File: examples/videostream-cli/cli.py [hunk 2]
```diff
@@ -143,15 +148,20 @@ def on_track(track):
...
 
+    # create save directory
+    if args.save_dir is not None:
+        if not os.path.exists(args.save_dir):
+            os.makedirs(args.save_dir)
+
     # create media source
     if args.play_from:
-        player = MediaPlayer(args.play_from, args.fps, args.save_dir)
+        player = MediaPlayer(args.play_from, args.enable_prediction, args.reference_update_freq, args.fps, args.save_dir)
```
Comment (vibhaa) [inline examples/videostream-cli/cli.py:158]: long line

Comment (vibhaa) [inline examples/videostream-cli/cli.py:158]: long line

File: examples/videostream-cli/cli.py [hunk 3]
```diff
@@ -143,15 +148,20 @@ def on_track(track):
...
     else:
         player = None
 
     # create media sink
     if args.record_to:
-        recorder = MediaRecorder(args.record_to, save_dir=args.save_dir)
+        recorder = MediaRecorder(args.record_to, enable_prediction=args.enable_prediction, output_fps=args.output_fps, save_dir=args.save_dir)
```
Comment (vibhaa) [inline examples/videostream-cli/cli.py:164]: long line

File: src/aiortc/contrib/media.py [hunk 4]
```diff
@@ -323,11 +321,13 @@ class MediaPlayer:
     :param options: Additional options to pass to FFmpeg.
     """
 
-    def __init__(self, file, fps=None, save_dir=None, format=None, options={}):
+    def __init__(self, file, enable_prediction=False, reference_update_freq=30, fps=None, save_dir=None, format=None, options={}):
```
Comment (vibhaa) [inline src/aiortc/contrib/media.py:324]: this line maybe too long too - can't tell from this view - please check

**Review by vibhaa — CHANGES_REQUESTED** [review]
Nothing major, standard spacing/long lines

File: examples/videostream-cli/cli.py (hunk 1 above)
Comment (pkarimib) [reply examples/videostream-cli/cli.py:138]: done

File: src/aiortc/contrib/media.py (hunk 4 above)
Comment (pkarimib) [reply src/aiortc/contrib/media.py:324]: shortened, thanks

**Review by vibhaa — APPROVED** [review]

**Pull request merged**